memory_model/src/mmap.rs

```diff
@@ -291,6 +291,35 @@ impl MemoryMapping {
         // overflow. However, it is possible to alias.
         std::slice::from_raw_parts_mut(self.addr, self.size)
     }
+    /// Uses madvise to tell the kernel to remove the specified range.  Subsequent reads
+    /// to the pages in the range will return zero bytes.
```

Comment:
This documentation can also benefit from improvement - `# Examples` section, parameter explanations, and a link to `madvise`'s documentation would help too (``[`madvise`](http://man7.org/linux/man-pages/man2/madvise.2.html)``)